scripts/update/main.go

```diff
@@ -129,14 +136,18 @@ func GetGoDeps(repo RepositoryDepsURL) []Repository {
 		//nolint
 		if strings.HasPrefix(dependency, "github.com") {
 			repourl := RepoURL{}
-			if err := repourl.Set(dependency); err == nil {
+			url := strings.Split(dependency, "/")
```

Comment:
can you create a helper function for this (with example comment on what this is doing). and use in both places.

Reply:
+1

Reply:
Agreed, I will do that. Thanks